tests/rest/client/v2_alpha/test_upgrade_room.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Half-Shot
```

Comment:
This isn't the copyright header I expected :yum:

Reply:
That's what I get for copying and pasting a file...